src/reduceRight.js

```diff
@@ -37,7 +37,7 @@ var _curry3 = require('./internal/_curry3');
 module.exports = _curry3(function reduceRight(fn, acc, list) {
   var idx = list.length - 1;
   while (idx >= 0) {
-    acc = fn(acc, list[idx]);
+    acc = fn(list[idx], acc);
     idx -= 1;
   }
   return acc;
```

Comment:
I think the comments should also be updated, for this one and for `reduce`, probably with examples that between them demonstrate how `reduce` and `reduceRight` are fundamentally different.  Perhaps something like:

```js
R.reduce((a, b) => a - b, 0, [1, 2, 3, 4]); //=> (((0 - 1) - 2) - 3) - 4) => -10
R.reduceRight((a, b) => a - b, 0, [1, 2, 3, 4]); //=> 0 - (1 - (2 - (3 - 4))) => 2
```